src/aktualizr_secondary/update_agent_file.cc

```diff
@@ -0,0 +1,62 @@
+#include "update_agent_file.h"
+#include "logging/logging.h"
+#include "uptane/manifest.h"
+
+// TODO: this is an initial version of a file update on Secondary
```

Comment:
Is this `TODO` still relevant?

Reply:
well, it depends on what quality we would like to have here. At least, I am planning to add more tests for this case (file update)

Reply:
Oh, no problem. It can stay! I was just checking.